dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchBar.vue

```diff
@@ -1,19 +1,22 @@
 <template>
   <MarginWrapper >
     <div class="grid">
-      <div class="col-8 text-left" v-if="!scrolled">
+      <div class="col-8 text-left">
         <span class="p-fluid">
           <span class="p-input-icon-left p-input-icon-right ">
             <i class="pi pi-search" aria-hidden="true" style="z-index:20; color:#958D7C"/>
             <i v-if="loading" class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index:20; color:#e67f3f"/>
             <i v-else aria-hidden="true"/>
             <AutoComplete
-                v-model="selectedCompany" :suggestions="filteredCompaniesBasic" @focus="focused" @focusout="unfocused"
-                @complete="searchCompany($event)" placeholder="Search company by name or PermID" inputClass="h-3rem" ref="autocomplete"
-                field="companyName" style="z-index:10" name="eu_taxonomy_search_input"
-                @keyup.enter="handleQuery" @item-select="handleItemSelect">
+                v-model="selectedCompany" :suggestions="autocompleteArrayDisplayed" :name="taxoSearchBarName"
+                ref="autocomplete" inputClass="h-3rem" field="companyName" style="z-index:10"
+                placeholder="Search company by name or PermID"
+                @focus="focused" @focusout="unfocused"
```

Comment:
Wofür wird das @ focus und @ focusout benötigt? Ich finde keine Referenz zu den dort hinterlegten Funktionen

Reply:
Die Methoden `focused()` und `unfocused()` wurden entfernt, daher werden die Events hier auch nicht mehr benötigt, das stimmt.

Ich wunder mich nur dass das hier in GitHub noch angezeigt wird, weil eigentlich hatte ich die gestern Mittag rausgeschmissen. Komisch.

Im Code sind sie auf jeden Fall weg. Pull gerne nochmal und schau es dir an um sicherzugehen.